mkdocs.yml

```diff
@@ -27,6 +28,7 @@ plugins:
 markdown_extensions:
   - pymdownx.highlight:
       anchor_linenums: true
+  - admonition
```

Comment:
What this for?

Reply:
!!! for highlight blocks https://squidfunk.github.io/mkdocs-material/reference/admonitions/